airflow/gcp/hooks/bigtable.py

```diff
@@ -276,3 +278,58 @@ def get_cluster_states_for_table(instance: Instance, table_id: str) -> Dict[str,
 
         table = Table(table_id, instance)
         return table.get_cluster_states()
+
+    def create_rowkey_regex_filter(self, regex):
+        """
+        A helper method to create a RowKeyRegexFilter subclass of RowFilter, which
+        will only match rows with keys that exactly match the regex string.
+        Returns a RowFilter.
+
+        :type regex: str
+        :param regex: The exact match regex string for the value we wish to find
+        """
+        return RowKeyRegexFilter(regex)
+
+
+    def delete_row(self, instance, table_id, row_key):
```

Comment:
Please add type annotations.
```suggestion
    def delete_row(self, instance: Instance, table_id: str, row_key: str) -> None:
```